src/prisma/schema.prisma

```diff
@@ -402,6 +415,32 @@ model StripeSubscription {
   @@index([spaceId])
 }
 
+model SpaceSubscriptionPayment {
+  id              String   @id @default(uuid()) @db.Uuid
+  paidTokenAmount String
+  spaceId         String?  @db.Uuid
+  space           Space?   @relation(fields: [spaceId], references: [id], onDelete: SetNull)
+  createdAt       DateTime @default(now())
+}
+
+model SpaceSubscriptionContribution {
+  id              String         @id @default(uuid()) @db.Uuid
+  spaceId         String?        @db.Uuid
+  space           Space?         @relation(fields: [spaceId], references: [id], onDelete: SetNull)
+  userId          String?        @db.Uuid
+  user            User?          @relation(fields: [userId], references: [id], onDelete: SetNull)
+  walletAddress   String
+  paidTokenAmount String
+  decentStatus    DecentTxStatus
+  decentError     Json?
+  decentTxHash    String?
+  decentChainId   Int?
+  decentPayload   Json
+  txHash          String?
+  chainId         Int?
+  createdAt       DateTime       @default(now())
```

Comment:
i would add an updatedAt field too (using prisma's built-in type so it is automatically set)